### ISSUE TITLE
2016 Talk 3: Wazim

### ISSUE THREAD

**Issue by ccshan** [wonks/ICFP_rehearsal_feedback#16]

**Comment by cgswords** [comment]
Mostly, a great talk. It was informative, and I felt like I really understood what you were doing! That was really cool! You probably want a laser pointer for the real talk to point at the slides.

Some nit-picky things (almost entirely slide-formatting things; the presentation itself was solid):
- The slides for introducing the initial language were incredibly crowded. I would remove the judgment rules and the distribution pictures; simply provide the implementation and invocations and describe the behavior.
- Add more space between the 'slide title' and the 'section title'; make it clear what we should really be paying attention to.
- Why \x . cx instead of just c ?
- Casually dropping the word 'continuation' to explain an integral is... confusing. Ah, the second slide clarified. I'd try to be stronger about that connection.
- Please use `&&` or `and` instead of `^` for and.
- The generalization of the density examples was a bit hard to follow. More directly (maybe formally) stating the induction and the cases would provide nice sign-posting.
- What are the emojis for? They clutter the slide; maybe strip them?
- I missed why we went to lists, or what the lists actually contain. Presumably the density; how do we interpret that result? Also, don't use Haskell's list comprehension syntax there because it's easy to confuse `[ .. | .. ]` from `[]` on a slide.
- `In The Paper` could be 3 slides, and broken out more. Also, actually hearing about applications would be swell.

**Comment by cderici** [comment]
- probably good idea to avoid more than one title for one slide
- too many information on the "Calculating probability densities" slide, it was somewhat hard for me to follow, also in the "Deriving density calculator".
- it may help to be consistent on showing lambda functions (with or without black background)
- it's unclear from the slide whether the `Exp e | Var v | Lev v e e ...` are already in the paper, or are those future work..

**Comment by ccshan** [comment]
Matt Might recommends a good laser pointer, but you might be able to find a brighter one. Avoid Bluetooth.
